src/enkf/enkf_obs_sensitivity.f90

```diff
@@ -48,18 +49,19 @@ module enkf_obs_sensitivity
 use convinfo, only: convinfo_read,init_convinfo
 use ozinfo, only: ozinfo_read,init_oz
 use radinfo, only: radinfo_read,jpch_rad,nusis,nuchan,npred
-use gridinfo, only: latsgrd,lonsgrd,nlevs_pres,npts
-use loadbal, only: indxproc,grdloc_chunk,numptsperproc,npts_max,kdtree_grid
+!use gridinfo_efsoi, only: latsgrd,lonsgrd,nlevs_pres,npts,id_u,id_v
+use loadbal!, only: indxproc,grdloc_chunk,numptsperproc,npts_max,kdtree_grid
```

Comment:
Is this necessary?  It's undesirable to include all variables/routines and rather use only what you need.

Reply:
Corrected